features/docs/gegeven-stap-definities-expressief.feature

```diff
@@ -0,0 +1,142 @@
+      |      | kind-1       | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr) VALUES($1,$2,$3,$4,$5)                                                                                          | 9999,0,0,K,000000036                        |
+      | 2    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *                                                     | 0                                           |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                        | 10000,0,0,P,000000024,P3                    |
+      |      | partner      | INSERT INTO public.lo3_pl_persoon(pl_id,volg_nr,burger_service_nr,geslachts_naam,relatie_eind_datum,relatie_eind_plaats,relatie_eind_land_code,persoon_type,stapel_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8,$9) | 10000,0,000000012,P2,gisteren - 1 jaar,0518,6030,R,0 |
+      |      | kind-1       | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr) VALUES($1,$2,$3,$4,$5)                                                                                          | 10000,0,0,K,000000036                       |
+      | 3    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *                                                     | 0                                           |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                        | 10001,0,0,P,000000036,P1                    |
+      |      | ouder-1-1    | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                        | 10001,0,0,1,000000012,P2                    |
+      |      | ouder-2-1    | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                        | 10001,0,0,2,000000024,P3                    |
+
+  Scenario: de persoon '[aanduiding]' met burgerservicenummer '[bsn]' is geadopteerd
+    Gegeven de persoon 'P1' met burgerservicenummer '000000036'
+    * is geadopteerd
```

Comment:
ik denk dat we een andere manier moeten bedenken om adoptie te beschrijven dan nu met stappen:
1. heeft een ouder 2 'P2' met burgerservicenummer '000000012'
2. is geadopteerd
3. zijn de volgende gegevens van ouder 'P2' gewijzigd

in de werkelijke wereld gebeurt dit (als de minderjarige daarvoor al is gemaakt met bsn 000000036):
1. ouder 2 is onbekend en vastgelegd als een puntouder
2. is geadopteerd door 'P2' met burgerservicenummer '000000012' als ouder 2

De eerste stap leidt tot:
```
      | stap | categorie | text                                                                                                                                        | values                          |
      | 1    | persoon   | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,geslachts_naam,familie_betrek_start_datum) VALUES($1,$2,$3,$4,$5,$6) | 9999,0,0,2,.,gisteren - 17 jaar |
```

Na de tweede stap zijn de inserts:
```
      | stap | categorie    | text                                                                                                                                                                        | values                                             |
      | 1    | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,geslachts_naam,familie_betrek_start_datum) VALUES($1,$2,$3,$4,$5,$6)                                 | 9999,0,1,2,.,gisteren - 17 jaar                    |
      | 2    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *                       | 0                                                  |
      | 2    | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                          | 10000,0,0,P,000000012,P2                           |
      | 3    | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam,geboorte_datum) VALUES($1,$2,$3,$4,$5,$6,$7)                        | 9999,0,1,P,000000036,P1,gisteren - 17 jaar         |
      | 3    | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam,geboorte_datum,akte_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8)             | 9999,0,0,P,000000036,P1,gisteren - 17 jaar,1AQ0100 |
      | 4    | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam,familie_betrek_start_datum,akte_nr) VALUES($1,$2,$3,$4,$5,$6,$7,$8) | 9999,0,0,2,000000012,P2,gisteren - 4 jaar,1AQ0100  |
      | 5    | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                          | 10000,0,0,K,000000036,P1                           |
```

waarbij (volgorde van stappen/inserts is willekeurig):
      - stap 1: ouderrelatie van de minderjarige naar puntouder wordt gemaakt. Door de tweede gegevenstap krijgt deze volg_nr 1.
      - stap 2: de persoon wordt gemaakt van de adopterende ouder
      - stap 3: in de bestaande minderjarige persoon insert wordt volgnummer opgehoogd naar 1. In een nieuwe insert met volg_nr 0 wordt aktenummer 1AQ0100 (adoptie) toegevoegd.
      - stap 4: de ouderrelatie van de minderjarige naar de adopterende ouder wordt toegevoegd. Dit is de actuele dus volg_nr is 0. De datum familierechtelijke betrekking kan je bijvoorbeeld gisteren - 4 jaar gebruiken (na de geboorte en voor de default scheidingsdatum). Het aktenummer is default 1AQ0100 (adoptie).
      - stap 5: de kindrelatie van de adopterende ouder naar de minderjarige wordt toegevoegd.